client/public/disclaimer.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Disclaimer</title>
+</head>
+
+<body>
+    <h1>Disclaimer</h1>
+    <!-- italicized -->
+    <p style="font-style: italic;">
+        Last updated: Jan 24, 2023
+    </p>
+    <p>
+        Lifeline assumes no responsibility for losses/damages of any nature whatsoever as a result of the use of our
+        service. Lifeline has made every attempt to ensure the accuracy and reliability of our product. However, we
+        cannot guarantee that the results will be fully as expected. Individual results may vary.
+    </p>
+    <p>
+        Errors in the processing interpretation have the chance of occurring due to the nature of the current
+        limitations in the logic.
+    </p>
+    <p>
+        Accuracy of interpretation is also dependent on the structure of the file provided. Our product is the most
+        accurate when grids/tables are used, however, accuracy may vary if deadlines are written differently.
```

Comment:
"...when grids/tables are used," - this wording just threw me off for a sec, should change to something like "...when the document uploaded contains/includes/makes use of grids or tables..."? Otherwise LGTM!